okhttp/src/main/java/okhttp3/OkHttpClient.kt

```diff
@@ -303,6 +312,8 @@ open class OkHttpClient internal constructor(
     internal var readTimeout: Int = 10000
     internal var writeTimeout: Int = 10000
     internal var pingInterval: Int = 0
+    internal var webSocketCompression: Boolean = false
+    internal var webSocketContextTakeover: Boolean = false
```

Comment:
this should be true by default?

Reply:
Same as about webSocketCompression above.